### PR TITLE
SOC-6232 : Wrong notification content when react to activity after me…

### PR DESCRIPTION
**Pull request by rdenarie**
…ntion user

Prior to this change, a notification can be sent, saying "An activity was updated" when a user like an activity.
This is due to the fact that ActivityManager.updateActivity is called in saveLike and deleteLike. So, updateActivity listeners are launched
This change add a broadcast parameter in updateActivity, to be able to update an activity without broadcasting the event.

### CODE REVIEW

File: component/core/src/test/java/org/exoplatform/social/core/manager/ActivityManagerTest.java [hunk 1]
```diff
@@ -689,27 +717,59 @@ public void testDeleteLike() throws Exception {
...
+
+    activityManagerSpy.saveLike(demoActivity, johnIdentity);
+
+    demoActivity = activityManagerSpy.getActivity(demoActivity.getId());
+    assertEquals("demoActivity.getLikeIdentityIds().length must return: 1", 1, demoActivity.getLikeIdentityIds().length);
+
+    activityManagerSpy.deleteLike(demoActivity, johnIdentity);
+
+    //we want the call twice, one for the saveLike, one for deleteLike
+    Mockito.verify(activityManagerSpy,Mockito.times(2)).updateActivity(Mockito.any(ExoSocialActivity.class),Mockito.eq(false));
```
Comment (ahamdi) [inline component/core/src/test/java/org/exoplatform/social/core/manager/ActivityManagerTest.java]: Why does it need to call updateActivity twic e ?

File: component/core/src/main/java/org/exoplatform/social/core/manager/ActivityManagerImpl.java [hunk 2]
```diff
@@ -492,8 +505,8 @@ public void saveComment(ExoSocialActivity existingActivity, ExoSocialActivity ne
     } else {
       if (previousMentions.length > 0) {
         String mentions = String.join(",", previousMentions);
-        Map<String, String> mentionsTemplateParams = newComment.getTemplateParams() != null ? newComment.getTemplateParams()
-                                                                                            : new HashMap<>();
+        Map<String, String> templateParams = newComment.getTemplateParams();
```
[the remark this answers is not in the document]
Comment (mdiha) [reply component/core/src/main/java/org/exoplatform/social/core/manager/ActivityManagerImpl.java]: Okay

File: component/core/src/test/java/org/exoplatform/social/core/manager/ActivityManagerTest.java (hunk 1 above)
Comment (mdiha) [reply component/core/src/test/java/org/exoplatform/social/core/manager/ActivityManagerTest.java]: Okay i will chack it

Comment (mdiha) [reply component/core/src/test/java/org/exoplatform/social/core/manager/ActivityManagerTest.java]: I think that this test case should be moved to org.exoplatform.social.notification.channel.template.LikeMailBuilderTest 
this test doesn't test the real case.

**Pull request merged**